### PR TITLE
Fix the 0x66 preffix handling  for JMP, Jcc CALL for intel

### PR DESCRIPTION
**Pull request by n80fr1n60**

### CODE REVIEW

**Comment by krystalgamer** [comment]
I already submitted a fix #1010 but for whatever reason the developers didn't answer me. I wish you the best of luck.

File: arch/X86/X86DisassemblerDecoder.c [hunk 1]
```diff
@@ -1930,7 +1931,23 @@ static int readImmediate(struct InternalInstruction *insn, uint8_t size)
...
+      if (insn->mode == MODE_64BIT && insn->isPrefix66 && (insn->opcode == 0xE8 || insn->opcode == 0xE9 || insn->opcode == 0xFF  || 
+			      (insn->opcodeType == TWOBYTE  &&  insn->opcode >= 0x80 && insn->opcode <= 0x8f )  )){
+        local_size = 4;
+        insn->immSize = 8;
+        insn->displacementSize = 4;
+    }
+
+	if (insn->mode == MODE_64BIT && insn->isPrefix66 && insn->opcode >=0x70 && insn->opcode <= 0x7f) {
+		local_size = 1;
+        insn->immSize = 8;
```
Comment (radare) [inline arch/X86/X86DisassemblerDecoder.c]: wrong indentation

File: my_builder.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+./make.sh
```
Comment (radare) [inline my_builder.sh]: unnecessary  and unrelated to the pr

**Comment by radare** [comment]
please resolve the conflicts and rebase to 4.1

**Comment by krystalgamer** [comment]
@radare why are you commenting 1 year later?
This PR does not fix the issue and the creator most definitely won't rebase this.

**Comment by radare** [comment]
Yeah, sometimes i also wonder why i care about things no one else do

**Comment by n80fr1n60** [comment]
**This PR does not fix the issue and the creator most definitely won't rebase this.** 
---> mind adding op codes where this does not work ? 
---> interesting , why do you know I will not rebase ???

**Comment by krystalgamer** [comment]
@n80fr1n60 No I won't add shit. By helping you it'd mean I'd be contributing to this garbage disassembler.

Lastly, due to your low github activity I suspected you had moved on, you can do whatever you want, but still I just wanted to tell one of the developers directly that pilling PR without answering is just plain retarded. After so long without answers and such low activity on this repo, it's clear the creator just wants to grow his e-peen.

Don't believe me? Just check the showcases page: http://www.capstone-engine.org/showcase.html
The creator goes around github asking for  credits left and right and doesn't do shit with others contributions. My project and x64dbg are still there even though both have dropped capstone a long time ago.

**Comment by pranith** [comment]
Can you please create a new PR on [libcapstone](https://github.com/libcapstone/libcapstone)?

**Comment by Rot127** [comment]
Thank you for the PR! I closed it because it is out of date. With the new [auto-sync update for v6](https://github.com/capstone-engine/capstone/blob/next/docs/cs_v6_release_guide.md) we made many changes to some main architectures and will do also to others.
This also changed the requirements we have now for new PRs.
 
If you still want to merge the changes, please rebase your fix onto the newest `next` branch and open a new PR.

**Pull request closed without merging**